server/src/test/java/io/crate/integrationtests/UpdateIntegrationTest.java

```diff
@@ -1134,4 +1137,33 @@ public void test_update_continues_on_error() {
         execute("update test set a = a + 98");
         assertThat(response).hasRowCount(1);
     }
+
+    private static final Logger LOGGER = LogManager.getLogger(UpdateIntegrationTest.class);
+
+    public void test_bulk_update_with_dynamic_adaption() throws Exception {
```

Comment:
So far, i am testing on a full integration test and checking the logs, which is not acceptable.